docs/pipfile.md

```diff
@@ -240,7 +240,7 @@ where the lock file was last updated, even on untrusted networks.
 We recommend designing CI/CD deployments whereby the build does not alter the lock file as a side effect.
 In other words, you can use `pipenv lock` or `pipenv upgrade` to adjust your lockfile through local development,
 the PR process and approve those lock changes before deploying to production that version of the lockfile.
-In other words avoid having your CI issue `lock`, `update`, `upgrade` `uninstall` or `install` commands that will relock.
+In other words avoid having your CI issue `lock`, `update`, `upgrade` `uninstall` commands that will relock.
 Note:  It is counterintuitive that `pipenv install` re-locks and `pipenv sync` or `pipenv install --deploy` does not.
```

Comment:
This seems contrary to other changes in this PR.

> It is counterintuitive that `pipenv install` re-locks and `pipenv sync` or `pipenv install --deploy` does not.

This should be updated accordingly.

Reply:
Good find; All set!